flow/project.py

```diff
@@ -3169,12 +3309,33 @@ def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
         :type ignore_conditions:
             :py:class:`~.IgnoreConditions`
         :yield:
-            All instances of :class:`~.JobOperation` jobs are eligible for.
+            All instances of :class:`~._JobOperation` jobs are eligible for.
         """
         for job in jobs:
             for op in self._job_operations(job, ignore_conditions):
                 yield op
 
+    @deprecated(deprecated_in="0.10", removed_in="0.13", current_version=__version__)
```

Comment:
It will be deprecated in `0.11`
```suggestion
    @deprecated(deprecated_in="0.11", removed_in="0.13", current_version=__version__)
```